sympy/stats/joint_rv.py

```diff
@@ -330,3 +330,12 @@ def marginalise_out(self, expr, rv):
 
     def __call__(self, *args):
         return self.pdf(*args)
+
+class StochasticProcess(Basic, NamedArgsMixin):
```

Comment:
Ingeriting from `Expr` would mean allowing arithmatic operations directly on the stochastic process instead of the indexed object. I am not sure if that would be correct.

Reply:
OK